types/simple-lru/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for simple-lru
+// Project: https://github.com/smagch/simple-lru
+// Definitions by: NN <https://github.com/NN--->
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare module "simple-lru" {
```

Comment:
This shouldn't be written as a top-level declare. Please see other definitions for how to do this correctly.

Reply:
I used same technique as other typings like https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/core-js/index.d.ts

```js
declare module "core-js" {
    export = core;
}
```

What is correct way to define ? Just remove the declare module and that's all ?
Thanks.

Reply:
Yeah, in this case you just have to remove the top and bottom lines of that snippet

Reply:
Thanks. I will update the PR.